include/envoy/common/platform.h

```diff
@@ -142,6 +143,9 @@ struct msghdr {
 #define SOCKET_ERROR_INVAL WSAEINVAL
 #define SOCKET_ERROR_ADDR_IN_USE WSAEADDRINUSE
 
+namespace platform {
+constexpr std::string_view null_device_path{"NUL"};
```

Comment:
absl::string_view? (fix include above)

Reply:
Actually, my comment may be out of date, @lizan are we reverting from absl to std for string_view with the adoption of c++17?

Reply:
Based on the latest comment I opted for std::string_view